WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanel.m

```diff
@@ -27,6 +29,30 @@ - (void)beginSession
 {
     [Mixpanel sharedInstanceWithToken:[WordPressComApiCredentials mixpanelAPIToken]];
     [self refreshMetadata];
+    [self flagIfUserHasSeenLegacyEditor];
+}
+
+- (void)flagIfUserHasSeenLegacyEditor
+{
+    NSUserDefaults *standardDefaults = [NSUserDefaults standardUserDefaults];
+    if ([standardDefaults boolForKey:CheckedIfUserHasSeenLegacyEditor]) {
+        return;
+    }
+    
+    NSInteger sessionCount = [[[[Mixpanel sharedInstance] currentSuperProperties] numberForKey:@"session_count"] integerValue];
```

Comment:
Since this line comes up twice in the code, I'd consider replacing it with a method such that: 

```objective-c
NSInteger sessionCount = [[[[Mixpanel sharedInstance] currentSuperProperties] numberForKey:@"session_count"] integerValue];
```

Becomes:

```objective-c
NSInteger sessionCount = [self getSessionCount];
```